tools/sotn_permuter/permuter_loader.py

```diff
@@ -255,8 +255,10 @@ def sotn_finalize_compile_command(cmdline: importer.List[str]) -> str:
                 for a in sys.argv
                 if not a.startswith("--settings") and not a.endswith(".c")
             ]
-            sys.argv[1] = sys.argv[1].split("/")[-1].rstrip(".s")
-            args.function = args.function.split("/")[-1].rstrip(".s")
```

Comment:
I keep doing this no matter how many times I tell myself it's stupid.  rstrip removes the characters from the right side of the string until it none of the characters match the last character.  It should be .replace(".s")

Reply:
I was a bit confused too. I thought `rstrip` stood for "reverse strip". I ended up using `endswith(".s")` with `[-2]` to reduce potential side effects, as we know `.s` is only found at the very end of the path